docs/release-notes/patch_checkpoints.rst

```diff
@@ -0,0 +1,7 @@
+:orphan:
+
+**Bug Fixes**
+
+-  Checkpoints: Checkpoints that are deleted through experiment deletion will not attempt
```

Comment:
Patch checkpoints is an implementation detail that I assume most users won't be aware of.

I would just mention the user behavior of tensorboards files not being deleted when experiments were deleted being fixed.